src/Components/Tooltip/Tooltip.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import "./Tooltip.css";
+
+export const ToolTip = ({setData,setIsDone}) => {
+  const [range, setRange] = useState({ x: "0px", y: "0px" });
+  const pos = { top: range.x, left: range.y };
```

Comment:
This needs to be changed to state in further update.

Reply:
it already took state from range